newIDE/app/src/Utils/Duration.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+export const formatDuration = (
```

Comment:
there is a function in `DateDisplay.js` called `secondsToMinutesAndSeconds()` used for the video tutorials duration.
Can this be put in common if the result is the same?

Reply:
I'll have a look, thanks!